### PR TITLE
Homepage testing feedback

### PR DESCRIPTION
**Pull request by mohitgusain**
#72 

1. Testing feedback
2. Ui changes

### CODE REVIEW

File: public/wp-content/themes/glific/front-page.php [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
+			<h3 class="text-white fz-28 leading-43 fz-md-36 font-heebo-regular"><?php echo $cover_section['content']; ?></h3>
+			<div class="d-flex flex-column flex-xl-row justify-content-md-end justify-content-xl-start pr-7 mt-10.5 mt-md-9.5 mt-xl-13.5">
 				<a class="glific-button-border bg-theme-secondary py-4 w-170 fz-18 leading-27 text-center text-white font-heebo-bold mr-md-8 text-decoration-none" href="<?php echo $cover_section['link_first']['url']; ?>"><?php echo $cover_section['link_first']['text']; ?></a>
-				<a class="glific-button-border bg-white  py-4 w-170 fz-18 leading-27 text-center text-theme-secondary font-heebo-bold mt-6 text-decoration-none mt-md-0" href="<?php echo $cover_section['link_second']['url']; ?>"><?php echo $cover_section['link_second']['text']; ?></a>
+				<a class="glific-button-border bg-white  py-4 w-170 fz-18 leading-27 text-center text-theme-secondary font-heebo-bold text-decoration-none mt-md-10 mt-10.5 mt-xl-0" href="<?php echo $cover_section['link_second']['url']; ?>"><?php echo $cover_section['link_second']['text']; ?></a>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/front-page.php]: @mohitgusain Please break the lengthier line where ever possible.
```suggestion
				<a class="glific-button-border bg-white  py-4 w-170 fz-18 leading-27 text-center text-theme-secondary font-heebo-bold text-decoration-none mt-md-10 mt-10.5 mt-xl-0"
				    href="<?php echo $cover_section['link_second']['url']; ?>">
				        <?php echo $cover_section['link_second']['text']; ?>
				</a>
```

File: public/wp-content/themes/glific/src/scss/style.scss [hunk 2]
```diff
@@ -14,10 +14,6 @@ Text Domain: coloredcow
 
 @import "assets/bootstrap/bootstrap";
 
-body {
-    @extend .mt-26, .mt-xl-35;
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/src/scss/style.scss:18]: intentionally?

File: public/wp-content/themes/glific/header.php [hunk 3]
```diff
@@ -73,4 +80,4 @@
 	</div>
 </header>
 
-<body>
+<body class="<?php echo !empty($banner_content) ? 'mt-28.5' : 'mt-20.5' ; ?> " >
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/header.php:83]: Possible to use Laravel blade format instead of using raw PHP?

**Review by Satendra-SR — APPROVED** [review]
@mohitgusain Minor changes required. Please fix and merge.

File: public/wp-content/themes/glific/src/scss/style.scss (hunk 2 above)
Comment (mohitgusain) [reply public/wp-content/themes/glific/src/scss/style.scss:18]: what ?. I have define  styling in the dom

File: public/wp-content/themes/glific/header.php (hunk 3 above)
Comment (mohitgusain) [reply public/wp-content/themes/glific/header.php:83]: Need to look into it.

**Pull request merged**